boa/boa-id-v1.jsonld

```diff
@@ -0,0 +1,35 @@
+{
+  "@context":
+  {
+    "@version": 1.1,
+    "bid": "https://sphereon-opensource.github.io/factom-vc-contexts/boa/boa-id-v1#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "schema": "http://schema.org/",
+
+    "BoaIdCredential" : "bid:BoaIdCredential",
+
+    "issueDate": {
+      "@id": "bid:issueDate",
+      "@type": "xsd:dateTime"
+    },
+    "validityDuration": {
+      "@id": "bid:validityDuration",
+      "@type": "schema:duration"
+    },
+    "name": {
```

Comment:
givenName, lastName, additionalName from schema.org?
I think it makes sense to already split these up now. Yes it deviates from a POC, but from a VC perspective it makes sense to distinguish them already and it is rather easy to display it. It is highly likely that the backend systems also have these split up already

Reply:
Yes, good point. I have split them up.